v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/TextIOToBigQuery.java

```diff
@@ -236,10 +236,8 @@ public interface Options
         optional = false,
         regexes = {".+:.+\\..+"},
         description = "Output table to write to",
-        helpText =
-            "The location of the BigQuery table in which to store your processed data. If you reuse"
-                + " an existing table, it will be overwritten.",
-        example = "your-project:your-dataset.your-table")
+        helpText = "The fully qualified BigQuery table.",
```

Comment:
The original description needs updates, but has more information than this one and is clearer. Maybe this should be:

The location of the BigQuery table to use to store the processed data. If you reuse an existing table, it is overwritten.